assets/src/design-system/theme/colors.js

```diff
@@ -0,0 +1,108 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// colors that are not subject to light or dark mode are stored in colors.
+// colors related to light or dark mode are nested in colors.light or colors.dark
+// it's important that the object structure be the same so that eventually we can flip from dark to light theme, vic versa
+
+const brandColors = {
```

Comment:
nit: any reason for `brandColors` vs `brand` here? not seeing why it's the only one with a `Colors` suffix

Reply:
other than just reading from figma and typing it - nope! brand is absolutely better, thank yo for noticing this inconsistency.